### ISSUE TITLE
hxcpp 3.2.1 tests are failing

### ISSUE THREAD

**Issue by Simn** [HaxeFoundation/hscript#70]
https://travis-ci.org/HaxeFoundation/hscript/jobs/281877857#L2798

```
Compiling group: runtime
g++ -D_CRT_SECURE_NO_DEPRECATE -DHX_UNDEFINE_H -c -fvisibility=hidden -O2 -fpic -fPIC -Wno-overflow -DHX_LINUX -DHXCPP_M64 -DHXCPP_VISIT_ALLOCS(haxe) -DHXCPP_API_LEVEL=321(haxe) -m64 -DHXCPP_M64 -I/home/travis/haxe/lib/hxcpp/3,4,188/include ... tags=[haxe]
 - src/hx/Boot.cpp 
 - src/hx/Anon.cpp 
 - src/hx/CFFI.cpp  [haxe,static]
 - src/hx/Date.cpp 
 - src/hx/gc/GcCommon.cpp  [haxe,gc]
 - src/hx/gc/Immix.cpp  [haxe,gc]
 - src/hx/gc/GcRegCapture.cpp  [haxe,gc]
 - src/hx/Hash.cpp 
 - src/hx/Interface.cpp 
 - src/hx/Lib.cpp  [haxe,static]
 - src/hx/Object.cpp 
 - src/hx/StdLibs.cpp  [haxe,static]
 - src/hx/Debug.cpp 
 - src/hx/Thread.cpp 
 - src/Array.cpp 
 - src/hx/Class.cpp 
 - src/Dynamic.cpp 
 - src/Enum.cpp 
 - src/Math.cpp 
 - src/String.cpp  [haxe,hxstring]
Link: Test
Error : Could not load module std@get_env__1
The command "haxe bin/build-cpp.hxml          && ./bin/Test" exited with 255.
```

@hughsando

**Comment by hughsando** [comment]
Is there any way to get a 3.2.1 binary for linux?  Because it seems to work ok on windows.

**Comment by andyli** [comment]
You may use the one at https://github.com/HaxeFoundation/haxe/releases/tag/3.2.1

**Comment by andyli** [comment]
Fixed by 5da267fb31458f4f1f4e820c3c3b02bcafa2115b
